lib/save-carts.js

```diff
@@ -28,6 +28,16 @@ module.exports = ({ appSdk }) => {
               })
             }
           })
+        } else if (available && completed === true) {
+          db.get('SELECT cart_id, store_id, customers_ids, created_at FROM ecomplus_cart_watch WHERE cart_id = ?', [_id], (err, row) => {
+            if (!err && row) {
+              db.run('DELETE FROM ecomplus_cart_watch WHERE cart_id = ?', [_id], err => {
+                if (!err) {
+                  logger.log(`> cart ${_id} deleted / status: completed`)
+                }
+              })
+            }
+          })
```

Comment:
```suggestion
        } else {
          db.run('DELETE FROM ecomplus_cart_watch WHERE cart_id = ?', [_id], err => {
            if (!err) {
              logger.log(`> cart ${_id} deleted / status: completed`)
            }
          })
```

Se `!available` o carrinho também deve ser deletado, só um else já é suficiente..
Não tem porque o SELECT antes do DELETE aí, só vai ser uma operação a mais 😬